### PR TITLE
Create bootboas.html

### PR DESCRIPTION
**Pull request by shadowninjazx**
Guide on how to boot soas. 
Assets can be uploaded to assets folder for website to show up properly. 

soaspic
boot1
boot2
boot3
boot4

### CODE REVIEW

**Comment by walterbender** [comment]
The master website has moved to https://github.com/samdroid-apps/www-sugarlabs/tree/sugary

File: bootsoas.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  Once you have your sugar stick, plug it in to a USB port on your computer.<br>
+    The picture on the right will show you what it should look like.
+  </p>
+  
+</div>
+
+<div class="grid">
+  <img class="col col-2-3" src="assets/boot2.jpg">
+  <p class="col col-1-3">
+  <h2>Boot Device Select(Windows)</h2>
```
Comment (samdroid-apps) [inline bootsoas.html]: Please but a space, and maybe use `<nobold>`  (or whatever it was on the inquiry page) for the (Windows).

Also, maybe say Some Windows 8+ Devices, b/c older windows computers use tranditional boot menu.

File: bootsoas.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    To boot from the USB, you will need to press "Shift" while clicking "restart" as shown in the picture.<br>
+    This will take you into an advanced boot tab.
+  </p>
+</div>
+
+<div class="grid">
+  <img class="col col-2-3" src="assets/boot3.jpg">
+  <p class="col col-1-3">
+  <h2>Select Boot from USB(Windows)</h2>
```
Comment (samdroid-apps) [inline bootsoas.html]: Same as Line 43

File: bootsoas.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    Once you are in the advanced boot tab, select "Use a Device".
+  </p>
+</div>
+
+<div class="grid">
+  <img class="col col-2-3" src="assets/boot4.jpg">
+  <p class="col col-1-3">
+  <h2>Select USB(Windows)</h2>
```
Comment (samdroid-apps) [inline bootsoas.html]: Same as line 43

File: bootsoas.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    Now select your USB drive to boot into Sugar OS.
+    
+  </p>
+</div>
+
+<div class="grid">
+<img class="col col-2-3" src="assets/boot5.jpg">
+  <p class="col col-1-3">
+  <h2>Boot Sugar(Multiboot Menu)</h2>
```
Comment (samdroid-apps) [inline bootsoas.html]: Space please.  Also, maybe Traditional Boot?

File: bootsoas.html [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  
+</div>
+
+
+
+
+<div class="grid grid-no-bottom-border">
+  <p class="col col-3-3">
+  <h2>Enjoy the Sweet OS</h2>
+    Once you selected to boot from your Sugar USB, you should be directed into the Sugar OS interface.<br>
```
Comment (samdroid-apps) [inline bootsoas.html]: I like the pun up in the title, but please say Sugar on a Stick here.

File: bootsoas.html [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    If you have problems during the boot process, you can check the sugarlabs<a href="https://socialhelp.sugarlabs.org/" target="_blank"> social help.</a>
```
Comment (samdroid-apps) [inline bootsoas.html]: SugarLabs [Social Help].

Maybe just the link around the []s?

Comment (samdroid-apps) [inline bootsoas.html]: But don't actually put [] in the text of course :)

File: bootsoas.html [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+</div>
+<p><a href="/">Return to main page</a></p>
+
+<!-- TODO:  Reflection.  Portfolio?
+<div class="grid">
+  <img class="col col-1-3" src="https://i.imgur.com/BxgYDcM.jpg">
+  <p class="col col-2-3">
+    <b>Reflect?  What activity?</b>
+  </p>
+</div>-->
```
Comment (samdroid-apps) [inline bootsoas.html]: You can get rid of this comment

**Comment by samdroid-apps** [comment]
Hi @shadowninjazx, great guide.  Few items of feedback:

*  Please base it off the sugary branch of samdroid-apps/www-sugarlabs.  You can download that by using a desktop git client, or by going:

    git clone https://github.com/samdroid-apps/www-sugarlabs
    cd www-sugarlabs
    git checkout sugary

this will let you use the header through the page inheritance, like with https://github.com/samdroid-apps/www-sugarlabs/blob/sugary/inquiry.html

*  Please use the sugar on a stick logo on the 1st image.  You could probably split it into 2 images:  the first with the computer and usb (as a png) and the seccond with the logo as an svg.  Or maybe you can keep them both as svg and just have 1 image!  Logo materials:   http://wiki.sugarlabs.org/go/Marketing_Team/Logo

* the 3rd image (you pressing shift over the restart button) is a bit unclear.  Can you have a picture of you pressing shift and a *screenshot* of your mouse over the restart button?  Plus your tiles are pretty nice, but I don't want to download a large image of them.

* It would be nice to have pictures for the traditional bios method.  Maybe 1 where it says "press F12 for boot menu" and 1 with the boot menu.  I'm happy to provide some if you need, or if you want to have some from many manufactures (Dell, Lenovo, maybe acer if my friend is nice).  But please add some pictures.

**Comment by shadowninjazx** [comment]
Hi, thanks for the feedback. 

I will apply the fix based on the samdroid-apps/www-sugar-labs html and assets. I think me messing around with the pull thing lead to the walterbender version.

**Comment by shadowninjazx** [comment]
So just to make sure
I will be making two guides of different boot options

1) Win 8 / Win 10 advanced boot menu, 
2) Traditional boot device select by F9 

I figured the Win 8 / Win 10 version is more user friendly so that is why I created it for people with WIn 8 /Win 10

**Comment by walterbender** [comment]
sounds like a plan...

On Wed, Dec 9, 2015 at 8:05 AM, shadowninjazx <notifications@github.com>
wrote:

> So just to make sure
> I will be making two guides of different boot options
>
> 1) Win 8 / Win 10 advanced boot menu,
> 2) Traditional boot device select by F9
>
> I figured the Win 8 / Win 10 version is more user friendly so that is why
> I created it for people with WIn 8 /Win 10
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/walterbender/www-sugarlabs/pull/14#issuecomment-163224051>
> .
>



-- 
Walter Bender
Sugar Labs
http://www.sugarlabs.org

**Comment by shadowninjazx** [comment]
Hi, updated on https://github.com/walterbender/www-sugarlabs/pull/15

**Pull request closed without merging**